### ISSUE TITLE
Integration of state of the art pose estimation: ViTPose. 

### ISSUE THREAD

**Issue by khanhha** [open-mmlab/mmpose#1389]
Hi awesome developers from MMPose. 
A new state of the art pose estimation research has published their code. MMPose is the leading open-source project in this field, so I am really looking forward for an integration of this research into MMPose. 
Thanks

**Comment by liqikai9** [comment]
Thanks for your interest in MMPose and your suggestion. We will look into it. You can also comment on the roadmap of MMPose [here](https://github.com/open-mmlab/mmpose/issues/9).

**Comment by Annbless** [comment]
Hi@khanhha, Thanks for your interest in ViTPose.

Hi, awesome developers from MMPose. Thanks for your fantastic work in MMPose. The framework is wonderful and helps greatly in our ViTPose and other pose works! 

We are willing to integrate ViTPose into MMPose and contribute to the community. Most of the modifications exist in the backbone folder (i.e., register a new backbone) and simpleheadbaseline.py (i.e., support a simpler decoder and add a parameter for control). A new optimizer constructor is also utilized in the mmcv_custom folder to support layer-wise learning rate decay for training. 

If there is anything that we could help with, please do not hesitate to contact us.

**Comment by ly015** [comment]
@Annbless Thank you very much for your help! It's inspiring to see that MMPose is facilitating great research works like ViTPose and bringing convenience to the community. And the contribution from the community is very much appreciated. We are looking forward to your PRs to integrate ViTPose into MMPose. Please let us know if there is anything we can help with.
